syncd/SelectablesTracker.cpp

```diff
@@ -0,0 +1,103 @@
+#include "SelectablesTracker.h"
+
+#include "swss/logger.h"
+
+using namespace syncd;
+
+bool SelectablesTracker::addSelectableToTracker(
+        _In_ swss::Selectable *selectable,
+        _In_ SelectableEventHandler *eventHandler)
+{
+    SWSS_LOG_ENTER();
+
+    if (selectable == nullptr)
+    {
+        SWSS_LOG_ERROR("Invalid Selectable:Selectable is NULL.");
+
+        return false;
+    }
+
+    int fd = selectable->getFd();
+    if (eventHandler == nullptr)
```

Comment:
move the check before line 20

Reply:
althoug he wanted to log fd number in error message, anyway please add empty line before if()